### PR TITLE
Use model constants in seed.rb and some refactor

### PR DESCRIPTION
**Pull request by emanreyalp**
Was there any reason why we used the aut.sch through http instead of https?
- I changed to the https

I did some changes in the seed file too:
- I added the missing ID-s to the group model and use it in the seed.
- I did some refactor in the seed for create users and them memberships and its posts.
- I complemented the seed with users and memberships. 

**Now we can use the application only with the seed file.**

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -17,28 +17,45 @@
...
-Membership.create({ group_id: 146, user_id: rvt_member.id })
-
-kb_user = User.create({ firstname: 'Srác', lastname: 'KBs', screen_name: 'kbs123' })
-Membership.create({ group_id: 1, user_id: kb_user.id })
+# Create leaders for the groups
+[
+  { firstname: 'Elnok', lastname: 'Rvt', screen_name: 'rvt_elnok', group_id: Group::RVT_ID },
+  { firstname: 'Elnok', lastname: 'Svie', screen_name: 'svie_elnok', group_id: Group::SVIE_ID },
+  { firstname: 'B. Martin', lastname: 'Kircsi', screen_name: 'kiraly96', group_id: Group::KIRDEV_ID },
+  # { firstname: 'Bálint Martin', lastname: 'Király' , screen_name: 'kiraly96', group_id: Group::KIRDEV_ID },
```
Comment (OmTheTurtle) [inline db/seeds.rb]: Why is this line commented out?

**Comment by OmTheTurtle** [comment]
@emanreyalp Merged the other PR about the seeds, please update this one.

File: db/seeds.rb (hunk 1 above)
Comment (emanreyalp) [reply db/seeds.rb]: I deleted both of lines and changed it to an anonymized user.

**Comment by emanreyalp** [comment]
I refreshed the commits and created a new one where I used the post type's ids in seed file.

**Pull request merged**